### DISCUSSION TITLE
Custom sort by an specified column

### DISCUSSION THREAD

**Discussion by JesusKT** [TanStack/table#2964] [Ideas]
Hi, i would like to know if there are any option to do an custom sort for an specified column

Also my problem is that my "value" of this column, is an icon ... so the default React-Table knowledge doesn't work well, because it doesn't group them like two different values.

`Header: t('some.critical'),
			id: 'critical',
			accessor: e => e.critical ? <i className="fas fa-exclamation-circle critical-icon" onClick={() => toggleCritical(e)}/> : <i className="fal fa-exclamation-circle" onClick={() => toggleCritical(e)}></i>
`

Will you see some approach for this?. I only need two values, true/false but with icon. I've been searching in doc, and another discussion for similar custom sort, but haven't found nothing that works for me.

I'm grateful for your time. Thanks in Advance!

**Comment by james-Infotrack** [comment]
you can pass you're own sortingFn in the column definition 
https://tanstack.com/table/v8/docs/api/features/sorting#sortingfn
